packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -368,9 +368,13 @@ class InputDecorator extends StatelessWidget {
   /// Defaults to false.
   final bool isEmpty;
 
-  /// The widget below this widget in the tree.
+  /// The widget below this widget in the tree.  Typically, this is an [EditableText] widget.
```

Comment:
I don't think that's true. In fact, of the two (non-test) cases where we create an InputDecorator in our code base, neither uses EditableText here. One uses a RepaintBoundary that contains an EditableText, and the other uses a Row to do fancy stuff.

I think most people using this directly will be doing so because they want to create a widget that matches a text field but isn't one.

Reply:
OK, reverted.